contributing.md

```diff
@@ -13,18 +13,33 @@ machine to develop Bazel and, when you've made a patch, how to submit it.</p>
 In general, we prefer contributions that fix bugs or add features (as opposed to
 stylistic, refactoring, or "cleanup" changes).
 
-Please check with us on the [dev
-list](https://groups.google.com/forum/#!forum/bazel-dev) before investing a lot
-of time in a patch. Meet Bazel maintainers and other contributors on
-[Slack](https://slack.bazel.build).
+### What issues can I work on?
+
+In general, if there is a Bazel team member assigned to an issue, someone is
+working on it. If not, you can use the priority level to as a rough guide.
+
+-   **P0**: someone from the Bazel team is be urgently working on it. If it is
+    not assigned yet, someone will be shortly.
+-   **P1**: someone is actively working on it, even if the assignee is not set.
+-   **P2**: a team knows they would like to get it done, but it is mixed with
+    work of other priorities. If you want to work on these, please check in the
+    issue first, to make sure someone has not already started on it but
+    forgotten to assign it to themselves.
+-   **P3/P4**: this might be worth doing, but no one is planning to work on it
+    at this time. We welcome community contributions for those.
 
 ### Patch Acceptance Process
 
 1. Read the [Bazel governance plan](governance.html).
-1. Discuss your plan and design, and get agreement on a GitHub issue or our
-   [mailing list](https://groups.google.com/forum/#!forum/bazel-dev).
-   Significant changes need a [design document](designs/index.html).
-1. Prepare a git commit that implements the feature. Don't forget to add tests.
+1. Significant changes need a [design review](designs/index.html). Before
+   sending your first PR you should read the design review process to
+   help you decide how to categorize the scope of your change.
+1. For significant changes, you should first discuss it on the
+   [developer mailing list](https://groups.google.com/forum/#!forum/bazel-dev).
+1. Discuss your plan and design, and get agreement on a GitHub issue.
```

Comment:
bazel-dev or GitHub issue, pick one? Unclear if I should start two threads of discussions or one.